dist/tools/genconfigheader/riotbuild-prefix.h.in

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2024 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    config_riotbuild    riotbuild.h: Preprocessor Constants to Query the Build System Configuration
+ * @ingroup     config
+ *
+ * The RIOT build system generates a header file `riotbuild.h` that is included
+ * in every C compilation unit by passing the `-include` flag to the C compiler.
+ * Hence, it provides a set of macros that are unconditionally available and
+ * can be used by C code to query the build system configuration.
+ * @{
+ */
+
+#ifdef DOXYGEN
+/**
+ * @brief   The used RIOT version as human readable string literal, e.g. for
+ *          printing to stdio or writing to a log.
+ */
+#define RIOT_VERSION "<YEAR_OF_RELEASE>.<MONTH_OF_RELEASE>-<POSTFIX>"
+
+/**
+ * @brief   The used RIOT version as machine readable number, e.g. for testing
```

Comment:
```suggestion
 * @brief   The used RIOT version as machine readable number, e.g., for testing
```

Reply:
IIRC, `@brief` may also only be one line.